web/src/components/PreflightResultErrors.tsx

```diff
@@ -0,0 +1,242 @@
+                Unable to automatically run preflight checks
+              </h2>
+            </div>
+            <p className="u-marginTop--10 u-marginBottom--10 u-fontSize--normal u-lineHeight--normal u-textColor--bodyCopy u-fontWeight--normal">
+              The Kubernetes RBAC policy that the Admin Console is running with
+              does not have access to complete the Preflight Checks. It’s
+              recommended that you run these manually before proceeding.
+            </p>
+            <p
+              className="replicated-link u-fontSize--normal u-marginBottom--10"
+              onClick={toggleShowErrorDetails}
+            >
+              {state.showErrorDetails ? "Hide details" : "Show details"}
+            </p>
+            {state.showErrorDetails && (
+              <div className="flex-column flex flex1 monaco-editor-wrapper u-border--gray">
+                <MonacoEditor
+                  language="bash"
+                  value={valueFromAPI}
+                  height="300px"
+                  options={{
+                    readOnly: true,
+                    contextmenu: false,
+                    minimap: {
+                      enabled: false,
+                    },
+                    scrollBeyondLastLine: false,
+                  }}
+                />
+              </div>
+            )}
+            <div className="u-marginTop--20">
+              <h2 className="u-fontSize--largest u-textColor--primary u-fontWeight--bold u-lineHeight--normal">
+                Run Preflight Checks Manually
+              </h2>
+              <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">
+                Run the commands below from your workstation to complete the
+                Preflight Checks.
+              </p>
+              {command ? (
+                <CodeSnippet
+                  language="bash"
+                  canCopy={true}
+                  onCopyText={
+                    <span className="u-textColor--success">
+                      Command has been copied to your clipboard
+                    </span>
+                  }
+                >
+                  {command}
+                </CodeSnippet>
+              ) : null}
+            </div>
+            <div className="u-marginTop--30 flex justifyContent--flexEnd">
+              <span
+                className="replicated-link u-marginLeft--20 u-fontSize--normal"
+                onClick={props.ignorePermissionErrors}
+              >
+                Proceed with limited Preflights
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {errorMsg && (
+        <ErrorModal
+          errorModal={displayErrorModal}
+          toggleErrorModal={toggleErrorModal}
+          err={errorTitle}
+          errMsg={errorMsg}
+          appSlug={selectedApp?.slug || ""}
```

Comment:
removed the `appSlug` from the props and added the `selectedApp` hook to this component.